src/main/resources/hudson/plugins/sonar/SonarGlobalConfiguration/config.jelly

```diff
@@ -20,9 +18,9 @@
           <f:entry title="${%ServerUrl}" description="${%ServerUrlDescr}">
             <f:textbox name="sonar.serverUrl" value="${inst.getServerUrl()}"/>
           </f:entry>
-                    
-          <f:entry title="${%ServerToken}" description="${%ServerTokenDescr}">
-            <f:password name="sonar.serverAuthenticationToken" value="${inst.getServerAuthenticationToken()}"/>
+
+          <f:entry title="${%ServerToken}" field="credentialsId">
```

Comment:
Is it required to remove the description attribute? And if yes, don't forget to clean resource bundles.

Reply:
No not at all, I didn't actually touch this file, it was from the previous PR.
I've restored it now

Screenshot of what it looks like:
![image](https://user-images.githubusercontent.com/21194782/52698670-e4e91f00-2f6b-11e9-8b5e-f871ff870d73.png)